draft-ietf-webtrans-http2.md

```diff
@@ -117,38 +118,48 @@ WebTransport servers are identified by an HTTPS URI as defined in {{Section
 4.2.2 of HTTP}}.
 
 When an HTTP/2 connection is established, both the client and server have to
-send a SETTINGS_ENABLE_WEBTRANSPORT setting in order to indicate that they
-both support WebTransport over HTTP/2.
+send a SETTINGS_ENABLE_WEBTRANSPORT setting to indicate that they both support
+WebTransport over HTTP/2.
+
+The server also needs to send a SETTINGS_WEBTRANSPORT_MAX_SESSIONS setting with
+a value greater than "0" to indicate the number of concurrent sessions it is
+willing to receive.
 
 A client initiates a WebTransport session by sending an extended CONNECT request
 {{!RFC8441}}. If the server accepts the request, a WebTransport session is
 established. The stream that carries the CONNECT request is used to exchange
 bidirectional data for the session. This stream will be referred to as a
-*CONNECT stream*.  The stream ID of a CONNECT stream, which will be referred to
+*CONNECT stream*.  The stream ID of a CONNECT stream, which will be referrred to
```

Comment:
```suggestion
*CONNECT stream*.  The stream ID of a CONNECT stream, which will be referrrrred to
```

Reply:
Fixed in [167cad5](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/58/commits/167cad5b8fbcc1df3093bf2ec83fe598bdf6886c)